scripts/webidl/convert.js

```diff
@@ -71,6 +71,19 @@ const transformer = new Webidl2js({
 
     if (isSimpleIDLType(idl.idlType, "DOMString") || isSimpleIDLType(idl.idlType, "USVString")) {
       const isUSV = isSimpleIDLType(idl.idlType, "USVString");
+      if (idl.name === "crossOrigin") {
+        const getKeyword = this.addImport("../helpers/cors-settings", "getKeyword");
+        return {
+          get: `
+            const value = ${implObj}.getAttributeNS(null, "${attrName}");
+            return ${getKeyword}(value);
+          `,
+          set: `
+            ${implObj}.setAttributeNS(null, "${attrName}", V);
+          `
+        };
+      }
+
```

Comment:
It looks like the `crossOrigin` property is currently marked as `[Reflect]` in the IDL files. Reflection of an enumerated attribute (like `crossorigin`) isn't currently implemented, so we shouldn't use reflection here.

The right fix would be to remove the `[Reflect]` and delete this part. Then, add `crossOrigin` getters and setters in the implementation classes that use `cors-settings`. To do so, you could also create a mixin class like this: https://github.com/jsdom/jsdom/blob/8921128ee1efe6737e3ea200959eb01fe75ff3aa/lib/jsdom/living/nodes/HTMLAnchorElement-impl.js#L11 https://github.com/jsdom/jsdom/blob/8921128ee1efe6737e3ea200959eb01fe75ff3aa/lib/jsdom/living/nodes/HTMLAnchorElement-impl.js#L46

Reply:
> The crossorigin attribute, valid on the `<audio>`, `<img>`, `<link>`, `<script>`, and `<video>` elements...